{
  "about": "https://jvndb.jvn.jp/ja/contents/2020/JVNDB-2020-009895.html",
  "title": "Oracle MySQL の MySQL Server における InnoDB に関する脆弱性",
  "link": "https://jvndb.jvn.jp/ja/contents/2020/JVNDB-2020-009895.html",
  "description": "Oracle MySQL の MySQL Server には、InnoDB に関する処理に不備があるため、可用性に影響のある脆弱性が存在します。",
  "identifier": "JVNDB-2020-009895",
  "references": [
    {
      "text": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-14791",
      "source": "CVE",
      "id": "CVE-2020-14791"
    },
    {
      "text": "https://nvd.nist.gov/vuln/detail/CVE-2020-14791",
      "source": "NVD",
      "id": "CVE-2020-14791"
    },
    {
      "text": "https://www.ipa.go.jp/security/vuln/scap/cwe.html",
      "id": "CWE-noinfo",
      "title": "情報不足(CWE-noinfo)"
    }
  ],
  "cpe": [
    {
      "text": "cpe:/a:oracle:mysql",
      "version": "2.2",
      "vendor": "オラクル",
      "product": "MySQL"
    }
  ],
  "cvss": [
    {
      "version": "2.0",
      "score": "3.5",
      "type": "Base",
      "severity": "Low",
      "vector": "AV:N/AC:M/Au:S/C:N/I:N/A:P"
    },
    {
      "version": "3.0",
      "score": "2.2",
      "type": "Base",
      "severity": "Low",
      "vector": "CVSS:3.0/AV:N/AC:H/PR:H/UI:N/S:U/C:N/I:N/A:L"
    }
  ],
  "date": "2020-12-11T12:33+09:00",
  "issued": "2020-12-11T12:33+09:00",
  "modified": "2020-12-11T12:33+09:00"
}
